main/main.script

```diff
@@ -136,19 +137,17 @@ function init(self)
 		movedex.init()
 		natures.init()
 		pokedex.init()
-		profiles.init()
-
-		flow.delay(1) -- Extra timeout for loading screen
-		flow.until_true( function() return not monarch.is_busy() end)
 
-		trainer.load()
-		storage.init()
 		dex.init()
 		items.init()
 		filters.init()
 		feats.init()
 		network.init()
+		profiles.init()
 
+		flow.delay(0.5) -- Extra timeout for loading screen
```

Comment:
Side note: I've been thinking about this loading screen. What we could do instead is:
* Measure the time after the loading screen appears but before init stuff happens
* flow.until_true( current time > initial time + 1 second)

That would make sure the loading screen is up for 1 second, but not more than that if initialization took a while.